client/src/models/Test/MultipleChoiceQuestion.ts

```diff
@@ -23,9 +24,11 @@ class MultipleChoiceQuestion extends Question {
         input[i].bodyFormat,
         input[i].correct
       );
+      if (input[i].correct) {
+        this.correctAnswer = i;
+      }
```

Comment:
Because there will be some multiple choice questions with multiple correct possible answers, we should instead use the isCorrect() method on an answer to determine whether an answer is a correct answer.